src/i18n/strings/en_EN.json

```diff
@@ -793,6 +793,10 @@
     "You can leave the beta any time from settings or tapping on a beta badge, like the one above.": "You can leave the beta any time from settings or tapping on a beta badge, like the one above.",
     "Beta available for web, desktop and Android. Some features may be unavailable on your homeserver.": "Beta available for web, desktop and Android. Some features may be unavailable on your homeserver.",
     "Your feedback will help make spaces better. The more detail you can go into, the better.": "Your feedback will help make spaces better. The more detail you can go into, the better.",
+    "Show all rooms in Home": "Show all rooms in Home",
+    "Show people in spaces": "Show people in spaces",
+    "If disabled, you can still add Direct Messages to Personal Spaces. If enabled, you'll automatically see everyone who is a member of the Space.": "If disabled, you can still add Direct Messages to Personal Spaces. If enabled, you'll automatically see everyone who is a member of the Space.",
+    "Show notification badges for DMs in Spaces.": "Show notification badges for DMs in Spaces.",
```

Comment:
@t3chguy nit: Pls can we update this to:

> Show notification badges on People in Spaces

... without the full stop at the end, and with more consistency with the other settings & the room list labels?

Reply:
So I think the concern here is it might make someone think that this impacts the `People` category within the room list rather than exclusively the space panel. 

Hence me picking `for` rather than `on`

Reply:
Gotcha. In that case then I think:

> Show notification badges for People in Spaces

Will avoid using _DMs_ inconsistently with _People_ while still making sense contextually when enabled.

Reply:
We already use `Direct Messages` in the microcopy one line above, but your call